back-end/sentiment-api/tests/auth_service.bal

```diff
@@ -0,0 +1,71 @@
+import ballerina/http;
```

Comment:
Can we rename this file as `mock_sts_endpoint`

Reply:
Addressed with c33cde2bebadeb76270aa8fa8add15e6a8ea6a62